pkg/build/gobuild.go

```diff
@@ -130,8 +129,8 @@ func (gbo *gobuildOpener) Open() (Interface, error) {
 func NewGo(ctx context.Context, dir string, options ...Option) (Interface, error) {
 	gbo := &gobuildOpener{
 		build: build,
-		sbom:  sbom,
 		dir:   dir,
+		sbom:  spdx("(none)"),
```

Comment:
Should this pass a real version value?

Reply:
So this is something I spent a bit of time futzing around with, so LMK if you have idea wrt plumbing, but basically the place we inject `Version` today is in the consuming `pkg/commands` package.  The way the CLI defaults are set up, this is always overridden with `WithSPDX(Version)` so we get `(devel)` at HEAD today, and presumably even better with releases.  However, I wanted the library default to be SPDX as well, and used `(none)` to distinguish.